scss/_layouts_application-panels.scss

```diff
@@ -1,16 +1,28 @@
 @mixin vf-l-application-panels {
   $header-padding-bottom: $sp-unit * 6;
 
-  .p-panel,
-  .p-panel--aside {
+  .p-panel {
     background: $colors--light-theme--background-default;
     color: $colors--light-theme--text-default;
+    max-width: 100%;
     min-height: 100%; // make sure it fills whole container even when has little content
 
     &.is-dark {
       background: $colors--dark-theme--background-default;
       color: $colors--dark-theme--text-default;
     }
+
+    &.is-wide {
+      width: 720px; // ~ col-8
+    }
+
+    &.is-medium {
+      width: 530px; // ~ col-6
+    }
+
+    &.is-narrow {
+      width: 340px; // ~ col-4
```

Comment:
What about using `vw` unit instead to be responsive on all screen sizes. Such as `70vw`, `50vw` and `30vw`?

Reply:
These are just placeholders for now, we are discussing different variants with @lyubomir-popov.
We discussed flexible vw / % values as well, but it's not something we currently use often in Vanilla.
Currently layout is usually proportional to font size (rems) and not to screen size. Using 'responsive' values such as 30vw for narrow panel would mean we will need to redefine it on different breakpoints, because 30vw on large screen will fit much more content than on small screen. And we should make sure they are more consistent.